arkane/encorr/decomp.py

```diff
@@ -87,27 +87,28 @@ def substruct_decomp(mol: Chem.Mol) -> List[List[int]]:
     cliques.extend(ssr)
 
     nei_list = [[] for _ in range(n_atoms)]
-    for i in range(len(cliques)):
-        for atom in cliques[i]:
+    for i, c in enumerate(cliques):
+        for atom in c:
             nei_list[atom].append(i)
 
     # Merge Rings with intersection > 2 atoms
-    for i in range(len(cliques)):
-        if len(cliques[i]) > 2:
-            for atom in cliques[i]:
+    # for i in range(len(cliques)):
```

Comment:
Is this left here as a helpful comment or as a way to disable old code? If it is just old code you can delete this